build.sbt

```diff
@@ -137,20 +43,32 @@ lazy val laws = crossProject(JSPlatform, JVMPlatform)
   .jsSettings(commonJsSettings)
   .jvmSettings(commonJvmSettings)
 
-lazy val lawsJVM = laws.jvm
-lazy val lawsJS = laws.js
-
 lazy val tests = crossProject(JSPlatform, JVMPlatform)
   .crossType(CrossType.Pure)
   .dependsOn(core, laws)
   .enablePlugins(NoPublishPlugin)
   .settings(name := "cats-mtl-tests")
-  .settings(libraryDependencies ++= Seq(
-    "org.typelevel" %%% "cats-testkit" % CatsVersion,
-    "org.typelevel" %%% "discipline-munit" % "1.0.9"))
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.typelevel" %%% "cats-testkit" % CatsVersion,
+      "org.scalameta" %%% "munit" % "0.7.29",
+      "org.typelevel" %%% "discipline-munit" % "1.0.9"))
   .jsSettings(commonJsSettings)
   .jvmSettings(commonJvmSettings)
-  .jsSettings(Test / scalaJSLinkerConfig ~= (_.withModuleKind(ModuleKind.CommonJSModule)))
 
-lazy val testsJVM = tests.jvm
-lazy val testsJS = tests.js
+lazy val unidocs = project
+  .in(file("unidocs"))
+  .enablePlugins(TypelevelUnidocPlugin)
+  .settings(
+    name := "cats-mtl-docs",
+    ScalaUnidoc / unidoc / unidocProjectFilter := inProjects(core.jvm, laws.jvm)
+  )
+
+lazy val docs = project
+  .in(file("site"))
+  .enablePlugins(TypelevelSitePlugin)
+  .settings(
+    tlFatalWarnings := false,
+    laikaConfig ~= (_.withRawContent)
+  )
+  .dependsOn(core.jvm)
```

Comment:
I forgot to mention, documentation for these plugins is available here:
https://typelevel.org/sbt-typelevel/site.html